hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -19,10 +19,10 @@
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "/sw/device/lib/dif/dif_aon_timer",
-  version:            "1.0.0",
+  version:            "2.0.0",
   life_stage:         "L1",
-  design_stage:       "D3",
-  verification_stage: "V2S",
+  design_stage:       "D2",
```

Comment:
@GregAC is there any reason not to go to D2S directly? i.e., any outstanding S reviews?
Otherwise we could go directly to D2S.